src/Layoffs/Layoffs.java

```diff
@@ -0,0 +1,64 @@
+package Layoffs;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Stack;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+public class Layoffs implements ActionListener {
+	private JFrame frame = new JFrame();
+	private JPanel panel = new JPanel();
+	private JTextField nameBox = new JTextField(15);
+	private JTextField layoffBox = new JTextField(15);
+	private JButton hireButton = new JButton();
+	private JButton layoffButton = new JButton();
+	private Stack<String> employees = new Stack<>();
+
+	public void createUI() {
+		frame.setVisible(true);
+		frame.add(panel);
+		frame.setSize(300, 100);
+		frame.setTitle("Big Business");
```

Comment:
That's cool, you gave your frame a name 👍

Reply:
yeah, that is pretty cool ⭐️ 🌟 ⭐️ 🌟 ⭐️